src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -527,6 +527,11 @@ def _forward(self, model_inputs, return_timestamps=False, generate_kwargs=None):
             generate_kwargs["return_timestamps"] = return_timestamps
             if return_timestamps == "word":
                 generate_kwargs["return_token_timestamps"] = True
+
+                stride = model_inputs.get("stride")
+                if stride is not None:
+                    generate_kwargs["num_frames"] = stride[0] // self.feature_extractor.hop_length
```

Comment:
If `stride` is None (i.e. we don't use the chunking algorithm) then `num_frames` is also `None`? We can set the default for `num_frames` in the `generation_config` to the value we'd use **without** any chunking

Reply:
`num_frames=None` should emulate the default behaviour: i.e., crop by 3000 frames (which is the same as just not cropping).

Reply:
In particular,

> We can set the default for num_frames in the generation_config

wouldn't matter since not cropping is the same as using the default crop of 3000.

Reply:
Ah I see - thanks for the clarification! Good for me in that case